data/alphabet.txt

```diff
@@ -0,0 +1,32 @@
+# Each line in this file represents the text (usually a single character, but
+# not required) associated with a numeric label.
```

Comment:
The comment

    usually a single character, but not required

should be changed to

    a single character

see my comment on `util/text.py`